R/dataProcessPlots.R

```diff
@@ -405,10 +507,14 @@ dataProcessPlots = function(
                             legend.size, label.color, cumGroupAxis, groupName,
                             lineNameAxis, yaxis.name)
       print(qc_plot)
+      plots <- c(plots, list(qc_plot))
```

Comment:
Let's also improve this while we make changes to plotly code. Looks like a leftover from old coding style

Reply:
Commit [better list handling](https://github.com/Vitek-Lab/MSstats/pull/107/commits/5c098d0c5d50d0300d8bca65672c0d6e9ec382e7)